aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,21 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.
```

Comment:
Wondering if its worth it to show a visual for this use case?

Reply:
yeah thinking maybe on the Forms page? maybe we can add to an existing example?

Reply:
I'll make a ticket for us to work on this separately on the forms page like we discussed.

Reply:
https://github.com/grommet/hpe-design-system/issues/2981